src/features/organisation-list/index.tsx

```diff
@@ -21,100 +26,126 @@ export interface IOrganization extends IFilterOptions {
-    services: 'trainingProvider',
-    IPCRegistered: 'yes',
-    lookingFor: 'fundingSupport',
+    focusesOn: "youthMentalWellness",
+    services: "trainingProvider",
+    IPCRegistered: "yes",
+    lookingFor: "fundingSupport",
     renderText: `focusesOn: 'youthMentalWellness',\nservices: 'trainingProvider',\nIPCRegistered: 'yes',\nlookingFor: 'fundingSupport',`,
-    verified: 'no',
-    index: 3
+    verified: "no",
+    index: 3,
   },
   {
-    focusesOn: 'ocd',
-    services: 'workshops',
-    IPCRegistered: 'no',
-    lookingFor: 'fundingSupport',
+    focusesOn: "ocd",
+    services: "workshops",
+    IPCRegistered: "no",
+    lookingFor: "fundingSupport",
     renderText: `focusesOn: 'ocd',\nservices: 'workshops',\nIPCRegistered: 'no',\nlookingFor: 'fundingSupport',`,
-    verified: 'pending',
-    index: 4
+    verified: "pending",
+    index: 4,
   },
   {
-    focusesOn: 'overallMentalWellbeing',
-    services: 'counselling',
-    IPCRegistered: 'yes',
-    lookingFor: 'fundingSupport',
+    focusesOn: "overallMentalWellbeing",
+    services: "counselling",
+    IPCRegistered: "yes",
+    lookingFor: "fundingSupport",
     renderText: `focusesOn: 'overallMentalWellbeing',\nservices: 'counselling',\nIPCRegistered: 'yes',\nlookingFor: 'fundingSupport',`,
-    verified: 'no',
-    index: 5
+    verified: "no",
+    index: 5,
   },
   {
-    focusesOn: 'antiStigma',
-    services: 'counselling',
-    IPCRegistered: 'no',
-    lookingFor: 'partnershipOpportunities',
+    focusesOn: "antiStigma",
+    services: "counselling",
+    IPCRegistered: "no",
+    lookingFor: "partnershipOpportunities",
     renderText: `focusesOn: 'antiStigma',\nservices: 'counselling',\nIPCRegistered: 'no',\nlookingFor: 'partnershipOpportunities',`,
-    verified: 'yes',
-    index: 6
+    verified: "yes",
+    index: 6,
   },
   {
-    focusesOn: 'eatingDisorder',
-    services: 'counselling',
-    IPCRegistered: 'yes',
-    lookingFor: 'fundingSupport',
+    focusesOn: "eatingDisorder",
+    services: "counselling",
+    IPCRegistered: "yes",
+    lookingFor: "fundingSupport",
     renderText: `focusesOn: 'eatingDisorder',\nservices: 'counselling',\nIPCRegistered: 'yes',\nlookingFor: 'fundingSupport',`,
-    verified: 'no',
-    index: 7
+    verified: "no",
+    index: 7,
   },
   {
-    focusesOn: 'antiStigma',
-    services: 'counselling',
-    IPCRegistered: 'no',
-    lookingFor: 'fundingSupport',
+    focusesOn: "antiStigma",
+    services: "counselling",
+    IPCRegistered: "no",
+    lookingFor: "fundingSupport",
     renderText: `focusesOn: 'antiStigma',\nservices: 'counselling',\nIPCRegistered: 'no',\nlookingFor: 'fundingSupport',`,
-    verified: 'pending',
-    index: 8
+    verified: "pending",
+    index: 8,
   },
-]
+];
 
 const OrganisationList: React.FC = () => {
+  // store organisation card data
+  const [orgList, setOrgList] = useState<Organisation[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
```

Comment:
@c6z3h can you please assist with fixing this? The query doesn't seem to update the local `orgList` variable even though I'm using `setOrgList`. Also the `useEffect` runs twice, not sure what's triggering the refresh

Reply:
It does update the orgList variable, perhaps the issue faced is why it does not render as such? That would have to do with line 141 -- rendering still uses `mockOrganisations` list, instead of `orgList`.

The issue with double render is problematic though. Happened to find the solution here, Ctrl+F for **React.Strict mode is on**:
* https://stackoverflow.com/questions/60618844/react-hooks-useeffect-is-called-twice-even-if-an-empty-array-is-used-as-an-ar

Besides this, this reminded me of a very useful / relevant video that visually explains how the event loop in JS works:
* https://www.youtube.com/watch?v=8aGhZQkoFbQ

Reply:
Oh I see, so the `orgList` variable shows up with data in the logs for you? I log the orglist after the update in line 127 but it shows up blank. 
![image](https://user-images.githubusercontent.com/32955712/220877409-145ae7ba-a88b-4e66-baa7-08032fdb5bef.png)

Yes, I have yet to update the card components so that it uses the data retrieved from firestore.

Thanks for the links, will review to try to resolve the double render.